### ISSUE TITLE
TS7006: Parameter 'e' implicitly has an 'any' type.

### ISSUE THREAD

**Issue by ClaytonHunt** [brandonroberts/angular-router-loader#105]
In version 0.8.3 of this package, a change was made to line 39 of the utils.js file.
```javascript
() => { throw({ loadChunkError: true }); }
```
was changed to:
```javascript
e => { throw({ loadChunkError: true, details: e }); }
```
I appreciate the intent of the modification, however, it is causing our tslint checks to fail with the error:
```
TS7006: Parameter 'e' implicitly has an 'any' type.
```
I'm not sure how we can resolve the problem we are seeing while maintaining the error reporting capabilities that you added. I'm open to some discussion about it and would even be willing to do any of the work that is needed to make a fix. Including validating it on our code.

Thank you for taking the time to review this issue report and for helping us in getting it resolved,
Clayton Hunt

**Comment by ClaytonHunt** [comment]
Upon a little further inspection, I see that the code in question is a string. I am going to make a small modification and submit a PR. Still open to discussion and would love for you to review the PR once submitted.

Again, Thanks for your time and development effort on this loader.
Clayton Hunt

**Comment by andyrichardson** [comment]
Can confirm - I just encountered this issue after updating.

**Comment by ClaytonHunt** [comment]
I had to delete node_modules and package-lock.json to get the downgrade fix to work locally.

**Comment by damienbod** [comment]
I still have this issue with 0.8.3

TS7006: Parameter 'e' implicitly has an 'any' type.

Will this be fixed, maybe version 0.8.4?

Greetins Damien

**Comment by RGVGreatCoder** [comment]
Maybe if you change your line of code from:
   e => { throw({ loadChunkError: true, details: e }); }
to:
   (e: any) => { throw({ loadChunkError: true, details: e }); }

**Comment by brandonroberts** [comment]
This fix has been published as `0.8.4`
